docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -1,128 +1,166 @@
+
+[role="screenshot"]
+image::images/profiling-setup-popup.png[]
+
+Click *Set up Universal Profiling* to configure data ingestion.
+
+NOTE: To configure data ingestion, you need elevated privileges, typically the `elastic` user.
+
+If you're upgrading from a previous version with Universal Profiling enabled, see the <<profiling-upgrade,upgrade guide>>.
+
+IMPORTANT: When upgrading, you must remove all existing profiling data.
+We still recommend upgrading as the latest version contains several improvements and new features.
+
+[discrete]
+[[profiling-install-host-agent]]
+== Install the host-agent
+
+The host-agent profiles your fleet. You need to install and configure it on every machine that you want to profile.
+The host-agent needs  `root` / `CAP_SYS_ADMIN` privileges to run.
+
+After clicking *Set up Universal Profiling* in the previous step, you'll see the instructions for installing the host-agent.
+You can also find these instructions by clicking the *Add data* button in the top-right corner of the page.
+
+The following is an example of the provided instructions for {k8s}:
+
+[role="screenshot"]
+image::images/profiling-k8s-hostagent.png[]
+
+[discrete]
+[[profiling-agent-config-notes]]
+=== Host-agent configuration notes
+
+Consider the following when configuring your host-agent:
+
+* The instructions in Kibana work well for testing environments. For production environments, we recommend setting an immutable version.
+
+* The host-agent versioning scheme is **not aligned with the {stack} version scheme**.
+
+* The OS packages downloaded from `releases.prodfiler.com` have a version in their file name.
+
+* You can find a list of container image versions in the
+https://container-library.elastic.co/r/observability/profiling-agent[Elastic container library repository].
+
+* For {k8s} deployments, the Helm chart version is already used to configure the same container image, unless
+overwritten with the `version` parameter in the Helm values file.
+
+[discrete]
+[[profiling-install-integration]]
+== Install the Universal Profiling Agent integration
+
+[discrete]
+[[find-apm-configuration]]
+=== Find APM configuration
+
+{ecloud} runs a hosted version of {integrations-server} that includes the APM integration.
+
+. In {kib}, navigate to **Fleet** > **Agent policies** and select the **Elastic Cloud agent policy**.
+
+. Click **Elastic APM** in the **Names** column.
+
+. Inside the **General** > **Server configuration** section, write down the value in the **URL** field.
+
+. Inside the **Agent authorization** > **Maximum number of API keys of Agent authentication** section, write down the value in the **Secret token** field.
+
+[discrete]
+[[add-integration]]
+=== Add integration
+
+. Navigate to **Management** > **Integrations**.
+
+. Toggle **Display beta integrations** to the on position in the left sidebar.
+
+. In the **Search for integrations** text field, enter `Universal`.
```

Comment:
Searching for `Universal` returns two results. `Universal Profiling Agent` and `Universal Profiling Symbolizer` (and soonish `Universal Profiling Collector`). From an outside perspective these integrations look identical besides the name. So I think it is important to point out the differences and maybe explain the reasoning?

Reply:
For the moment, I added an exact search term so that only one result appears. I agree we should add an explanation for all of our integrations, but perhaps it should be in another section where it could get a more thorough explanation.